packages/block-library/src/paragraph/edit.js

```diff
@@ -246,7 +246,7 @@ class ParagraphBlock extends Component {
 					onReplace={ this.onReplace }
 					onRemove={ () => onReplace( [] ) }
 					aria-label={ __( 'Empty block; type text or press the forward slash key to insert a block' ) }
```

Comment:
should "type text" be replaced with "start writing"?

Reply:
For a screen reader I actually think "type text" is better than "start writing".